app/components/Inspector.js

```diff
@@ -1,34 +1,68 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import { Card } from 'antd';
+import { Card, Icon, Button, Spin } from 'antd';
 import Screenshot from './Inspector/Screenshot';
 import SelectedElement from './Inspector/SelectedElement';
 import Source from './Inspector/Source';
 import InspectorStyles from './Inspector.css';
 
 export default class Inspector extends Component {
 
+  constructor () {
+    super();
+    this.didInitialResize = false;
+  }
+
   componentWillMount () {
+    const chromeHeight = 48;
+    if (!this.didInitialResize) {
+      // resize width to something sensible for using the inspector on first run
+      window.resizeTo(1080, window.clientHeight - chromeHeight);
```

Comment:
This isn't when the BrowserWindow is created; this is when the router navigates from `#/session` to `#/inspector`. Regardless of how big the Session window was, or whether the user made it a random size, I want the inspector to start at a reasonable width (so that the screenshot and all the columns are nicely visible)